src/open_inwoner/accounts/views/cases.py

```diff
@@ -371,6 +380,51 @@ def get_case_document_files(self, case) -> List[SimpleFile]:
             )
         return documents
 
+    def get_information_object_type_choices(self) -> List[str]:
```

Comment:
I think in django 'choices' are generally the combination of values & labels (eg: the list of tuples). Then you also don't have to do additional noisy work where you'd want to use them (eg: in get_form_kwargs/get_context etc).

For now its fine as this will be refactored anyway.